language/Russian/strings.po

```diff
@@ -6371,17 +6390,17 @@ msgstr ""
 #: plugins/timer/init.lua:32
 msgctxt "plugin-timer"
 msgid "Wall clock"
-msgstr ""
+msgstr "Настенные часы"
 
 #: plugins/timer/init.lua:32
 msgctxt "plugin-timer"
 msgid "Emulated time"
-msgstr ""
+msgstr "Время эмуляции"
 
 #: plugins/timer/init.lua:37
 msgctxt "plugin-timer"
 msgid "Reference"
-msgstr ""
+msgstr "Ссылка"
```

Comment:
These messages are for setting the reference clock for measuring elapsed time.  “Wall clock” refers to a conceptual stable clock for measuring elapsed real time.  “Emulated time” refers to time from the point of view of the emulated system (so it’s affected by using the “fast forward” feature, and when emulation doesn’t run at full speed).  Are these translations appropriate?